src/lib.rs

```diff
@@ -188,6 +188,8 @@ pub struct InitOptions {
 pub fn init(options: &InitOptions) -> Result<ZWaveManager,()> {
     let mut ozw_options = try!(options::Options::create("./config/", "", "--SaveConfiguration true --DumpTriggerLevel 0 --ConsoleOutput false"));
 
+    try!(options::Options::add_option_int(&mut options, "PollInterval", 60000)); //enable polling each 60 seconds
+    try!(options::Options::add_option_bool(&mut options, "IntervalBetweenPolls", true));
```

Comment:
> This can also be set in the options.xml file.

Yes but we'll want to make it possible for the user to control this from the UI, I think ?

> I also noticed that the things we have in the Options::create call can also be added using add_option_xxx which is probably a bit more intuitive

Agreed; initially I just used the same lines than what is in the example programs.